src/tests/test_national.py

```diff
@@ -255,3 +256,97 @@ def test_read_truth_national_gsp(db_session, api_client):
     r_json = response.json()
     assert len(r_json) == 3
     _ = [GSPYield(**gsp_yield) for gsp_yield in r_json]
+
+
+def test_is_fake_national_all_available_forecasts(
+    monkeypatch, db_session, api_client, pytest_print=None
+):
+    """### Test FAKE environment for all GSPs are populating
+    with fake data.
+
+    #### Parameters
+    - **pytest_print**: If you'd like to inspect the forecast values
+    in the FAKE environment, please use 'pytest -s' in the CLI to inspect
+    the values. Using the -s switch should enable the print statement.
+    Otherwise can use 'pytest -v' and it will run the tests as normal and
+    suppress the verbose print statements.
+    """
+
+    monkeypatch.setenv("FAKE", "1")
+    assert is_fake() == 1
+    # Connect to DB endpoint
+    response = api_client.get("/v0/solar/GB/national/forecast")
+    assert response.status_code == 200
+
+    gsp_ids = [int(gsp_id) for gsp_id in range(GSP_TOTAL)]
+
+    forecasts = make_fake_forecasts(
+        gsp_ids=gsp_ids,
+        session=db_session,
+        forecast_values=None,
+        add_latest=True,
+        historic=True,
+        model_name="fake_model",
+    )
+    db_session.add_all(forecasts)
+    db_session.commit()
```

Comment:
Please remove, we dont need to add these in the test, they should be add under `is_fake` in national.py

Reply:
I've removed the calls to the DB in the test cases. Could you review, and let me know if I've done it correctly, please?